src/index.html

```diff
@@ -7,7 +7,306 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Hello Mate Academy</h1>
+    <header class="header">
+      <span class="header__logo">na<br>mu</span>
+      <div class="header__background">
+        <div class="header__art art">
+          <h1 class="art__title">
+            ХУДОЖ<span class="art__transparant-title">НІЙ</span><br> МУЗЕЙ
+          </h1>
+          <button class="art__btn">КВИТКИ<span data-icon=""></span></button>
+        </div>
+      </div>
+
+      <input
+        type="checkbox"
+        class="header__toggler"
+        id="menu-toggler"
+        >
+      <label for="menu-toggler" class="header__toggler-label">
+        <span class="header__toggler-menu"></span>
+      </label>
+
+      <nav class="header__menu menu">
+        <ul class="menu__list">
+          <li class="menu__item">
+            <a href="#event" class="menu__link">Події</a>
```

Comment:
Перенось текст на новий рядок, так простіше прочитати
```suggestion
            <a href="#event" class="menu__link">
              Події
            </a>
```